exercises/knapsack/knapsack_test.py

```diff
@@ -2,63 +2,102 @@
 
 from knapsack import maximum_value
 
-
 # Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
 
-class ChangeTest(unittest.TestCase):
+
+class KnapsackTest(unittest.TestCase):
     def test_no_items(self):
-        self.assertEqual(maximum_value(100, []), 0)
+        self.assertEqual(maximum_value(100, {}), 0)
```

Comment:
I know the canonical data includes a `{}` for an "empty" value, but we should continue to return an empty list, _not_ an empty dict, just to be internally consistent.

Reply:
Changes made!